src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -143,6 +128,23 @@ public Task InitializeAsync(CancellationToken cancellationToken)
             });
         }
 
+        public async Task OnRenderCompleted(long renderId, string errorMessageOrNull)
```

Comment:
Please add some comments here about the expected usage of this method, and its error handling model.

Please move this somewhere after the lifecycle methods. Right now this is right between `InitializeAsync` and `DisposeAsync`.

Reply:
done